runtime/src/main/java/io/quarkus/jsch/runtime/JSchProxyConfig.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.jsch.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigDocDefault;
+import io.quarkus.runtime.annotations.ConfigGroup;
+
+@ConfigGroup
+public interface JSchProxyConfig {
```

Comment:
Can we move this as an inner interface in `JSchRuntimeConfig`?

Reply:
It is used in `JSchRuntimeConfig` and also in `JSchSessionRuntimeConfig`

I move it anyway ?

Reply:
Move both to inner interfaces in `JSchRuntimeConfig`